### PR TITLE
Image tags

### PR DESCRIPTION
**Pull request by khushboo18**
Closes 5 - Image tags
---
- [x] Form field to add tags
- [x] Tags are displayed with the image once saved
- [x] On the index page, each image is displayed with associated tags
- [x] acts-as-taggable-on gem used to provide tagging

### CODE REVIEW

File: app/models/article.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 require 'httparty'
 
 class Article < ApplicationRecord
+  acts_as_taggable_on :tags
```
Comment (steventhan) [inline app/models/article.rb:5]: Can you add more tests for `Article` model? The test should cover model functionalities defined in this file

File: test/controllers/articles_controller_test.rb [hunk 2]
```diff
@@ -28,4 +28,16 @@ class ArticlesControllerTest < ActionDispatch::IntegrationTest
 
     assert_redirected_to article_url(Article.last)
   end
+
+  test 'add tags to image' do
```
Comment (steventhan) [inline test/controllers/articles_controller_test.rb:32]: Can you test for cases in which controller rejects the request as well?

Comment (khushboo18) [reply test/controllers/articles_controller_test.rb:32]: In what scenarios would the controller reject tagging requests?

Comment (steventhan) [reply test/controllers/articles_controller_test.rb:32]: One case I can think of is when the `:article` has more than the 3 permitted fields: `:link, :text, :tag_list`

**Comment by steventhan** [comment]
LGTM 🚢

**Pull request merged**